tests/test-runner/bin/zts-report.py.in

```diff
@@ -263,13 +263,50 @@ if sys.platform.startswith('freebsd'):
         'cli_root/zpool_import/zpool_import_012_pos': ['FAIL', known_reason],
         'delegate/zfs_allow_003_pos': ['FAIL', known_reason],
         'inheritance/inherit_001_pos': ['FAIL', 11829],
-        'resilver/resilver_restart_001': ['FAIL', known_reason],
         'pool_checkpoint/checkpoint_big_rewind': ['FAIL', 12622],
         'pool_checkpoint/checkpoint_indirect': ['FAIL', 12623],
+        'resilver/resilver_restart_001': ['FAIL', known_reason],
         'snapshot/snapshot_002_pos': ['FAIL', '14831'],
     })
 elif sys.platform.startswith('linux'):
     maybe.update({
+	'bclone/bclone_crossfs_corner_cases.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_crossfs_corner_cases_limited.ksh':
+            ['SKIP', cfr_cross_reason],
+	'bclone/bclone_crossfs_data.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_crossfs_embedded.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_crossfs_hole.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_diffprops_all.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_diffprops_checksum.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_diffprops_compress.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_diffprops_copies.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_diffprops_recordsize.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_prop_sync.ksh': ['SKIP', cfr_cross_reason],
+	'bclone/bclone_samefs_corner_cases.ksh': ['SKIP', cfr_reason],
+	'bclone/bclone_samefs_corner_cases_limited.ksh': ['SKIP', cfr_reason],
+	'bclone/bclone_samefs_data.ksh': ['SKIP', cfr_reason],
+	'bclone/bclone_samefs_embedded.ksh': ['SKIP', cfr_reason],
+	'bclone/bclone_samefs_hole.ksh': ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_copyfilerange':
+            ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_copyfilerange_cross_dataset':
+            ['SKIP', cfr_cross_reason],
+        'block_cloning/block_cloning_copyfilerange_fallback':
+            ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_copyfilerange_fallback_same_txg':
+            ['SKIP', cfr_cross_reason],
+        'block_cloning/block_cloning_copyfilerange_partial':
+            ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_cross_enc_dataset':
+            ['SKIP', cfr_cross_reason],
+        'block_cloning/block_cloning_disabled_copyfilerange':
+            ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_lwb_buffer_overflow':
+            ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_replay':
+            ['SKIP', cfr_reason],
+        'block_cloning/block_cloning_replay_encrypted':
+            ['SKIP', cfr_reason],
```

Comment:
I believe you need to the `.ksh` extension here.  These aren't being recognized as SKIPS.

```
Tests with results other than PASS that are unexpected:
    SKIP bclone/bclone_crossfs_corner_cases_limited (expected PASS)
    SKIP bclone/bclone_crossfs_data (expected PASS)
    SKIP bclone/bclone_crossfs_embedded (expected PASS)
    SKIP bclone/bclone_crossfs_hole (expected PASS)
    SKIP bclone/bclone_diffprops_all (expected PASS)
    SKIP bclone/bclone_diffprops_checksum (expected PASS)
    SKIP bclone/bclone_diffprops_compress (expected PASS)
    SKIP bclone/bclone_diffprops_copies (expected PASS)
    SKIP bclone/bclone_diffprops_recordsize (expected PASS)
    SKIP bclone/bclone_prop_sync (expected PASS)
    SKIP bclone/bclone_samefs_corner_cases_limited (expected PASS)
    SKIP bclone/bclone_samefs_data (expected PASS)
    SKIP bclone/bclone_samefs_embedded (expected PASS)
    SKIP bclone/bclone_samefs_hole (expected PASS)
```

The style checker also has [some formatting complaints]((https://github.com/openzfs/zfs/actions/runs/7294586309/job/19879696271?pr=15631) to address.  Then I _think_ this should be ready to go.